CMakeLists.txt

```diff
@@ -74,6 +74,19 @@ ELSE()
     target_link_libraries(Hyprland xcb)
 ENDIF(NO_XWAYLAND MATCHES true)
 
+IF(NO_SYSTEMD MATCHES true)
+    message(STATUS "SYSTEMD supoort is diabled...")
+ELSE()
+    message(STATUS "SYSTEMD support is requested (NO_SYSTEMD not defined) checking deps...")
+    pkg_check_modules(LIBSYSTEMD libsystemd)
+    IF(LIBSYSTEMD_FOUND)
+        add_definitions( -DUSES_SYSTEMD )
+        target_link_libraries(Hyprland "${LIBSYSTEMD_LIBRARIES}")
```

Comment:
what's LIBSYSTEMD_LIBRARIES?

Reply:
whatever `pkg-config libsystemd --libs` finds without the `-l`

Reply:
alr I'll trust you on that one steve